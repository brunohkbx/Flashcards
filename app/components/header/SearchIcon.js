import React from 'react';
import { TouchableOpacity } from 'react-native';
import { MaterialIcons } from '@expo/vector-icons';
import { white } from '../../config/theme';

const SearchIcon = () => {
  return (
    <TouchableOpacity style={{ marginHorizontal: 10 }} onPress={() => {}}>
      <MaterialIcons
        name="search"
        size={24}
        color={white}
      />
    </TouchableOpacity>
  );
}


export default SearchIcon;
