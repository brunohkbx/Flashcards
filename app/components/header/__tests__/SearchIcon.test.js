import React from 'react';
import { shallow } from 'enzyme';
import SearchIcon from '../SearchIcon';

describe('SearchIcon', () => {
  const setup = () => {
    const wrapper = shallow(<SearchIcon />);

    return {
      wrapper
    }
  }

  it('renders properly', () => {
    const { wrapper } = setup();

    expect(wrapper).toMatchSnapshot();
  });
});
