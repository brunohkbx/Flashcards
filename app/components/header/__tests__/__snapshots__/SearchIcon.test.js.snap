// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchIcon renders properly 1`] = `
<TouchableOpacity
  activeOpacity={0.2}
  onPress={[Function]}
  style={
    Object {
      "marginHorizontal": 10,
    }
  }
>
  <Icon
    allowFontScaling={false}
    color="#ffffff"
    name="search"
    size={24}
  />
</TouchableOpacity>
`;
